cockatrice/src/window_main.cpp

```diff
@@ -494,16 +499,20 @@ void MainWindow::activateError()
 
 void MainWindow::socketError(const QString &errorStr)
 {
-    QMessageBox::critical(this, tr("Error"), tr("Socket error: %1").arg(errorStr));
+    QMessageBox::critical(this, tr("Error"), tr("Socket error:") + QString(" %1").arg(errorStr));
     actConnect();
 }
 
 void MainWindow::protocolVersionMismatch(int localVersion, int remoteVersion)
 {
     if (localVersion > remoteVersion)
-        QMessageBox::critical(this, tr("Error"), tr("You are trying to connect to an obsolete server. Please downgrade your Cockatrice version or connect to a suitable server.\nLocal version is %1, remote version is %2.").arg(localVersion).arg(remoteVersion));
+        QMessageBox::critical(this, tr("Error"), tr("You are trying to connect to an obsolete server. Please downgrade your Cockatrice version or connect to a suitable server." + QString("<br>")
+                                                    + tr("Local version:") + QString(" %1<br>").arg(localVersion)
+                                                    + tr("Remote version:") + QString(" %1").arg(remoteVersion));
```

Comment:
`Local version` is the version of the client --> Cockatrice?
`Remote version` is the version the server is running --> Servatrice?

So go with `Server version:` for clarity?